docs/tutorials/helloworldintro.md

```diff
@@ -16,23 +21,27 @@ This "Hello World" guide walks you through creating a project, installing the ML
 This tutorial requires an MLAPI-supported version of Unity (2019.4+).
 :::
 
- ![Open Unity create new project](/img/openunity.gif)
+<iframe src="https://www.youtube.com/embed/NsfwlWaZ0ng?playlist=NsfwlWaZ0ng&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
 
 ## Import MLAPI Package via URL
 
 See the [Install MLAPI](../migration/installation.md) guide to install the MLAPI package.
 
 ## Create the Basic Components
 
+In this section we will create the basic building blocks of a multiplayer game
+
+### Creating Network Manager and selecting the Transport
+
 1. Right click in the Hierarchy tab of the Main Unity Window.
 1. Select **Create Empty**.
 1. Rename the GameObject **NetworkManager**.
    
-    ![create gameobject](/img/creategameobject.gif) 
   :::tip
   We renamed the GameObject because:
     * It makes it  easier to refer to later.
-    * There is one and only one **NetworkManager**, this is the object that contains the  `NetworkManager` component.
+    * There should only be one **NetworkManager**, this is the object that contains the  `NetworkManager` component.
```

Comment:
Just to capture the nuance of this...MLAPI currently simply won't work right if you happen to try and create more than one NM.

Reply:
Added a trailing line to this to reinforce the message

You may get unexpected results if you create more than one **NetworkManager**.